invokeai/app/services/shared/invocation_context.py

```diff
@@ -557,6 +557,24 @@ def sd_step_callback(self, intermediate_state: PipelineIntermediateState, base_m
             is_canceled=self.is_canceled,
         )
 
+    def flux_step_callback(self, intermediate_state: PipelineIntermediateState) -> None:
```

Comment:
Should we label this as deprecated, given the plan to replace it (https://github.com/invoke-ai/InvokeAI/pull/6715)? Feels a little weird to introduce something new to the API in a deprecated state though...

Reply:
I think setting it to deprecated can just be part of the PR that's deprecating it